src/safe_apps/migrations/0007_safeapp_access_policy.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2.9 on 2021-12-29 21:36
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("safe_apps", "0006_safeapp_chain_ids_big_int"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Client",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "url",
+                    models.CharField(
+                        help_text="The domain URL client is hosted at",
+                        max_length=255,
+                        unique=True,
+                        validators=[
+                            django.core.validators.RegexValidator(
+                                "^(https?:\\/\\/)?(www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\/?$",
```

Comment:
This regex is different than the one set in the model 🤔 

Maybe it's better to recreate this migration:

```shell
python src/manage.py migrate safe_apps 0006
rm src/safe_apps/migrations/0007_safeapp_access_policy.py
python src/manage.py makemigrations
```

Reply:
It's the same, isn't it? I recreated the migration before pushing and did it once again but the output is the same

Reply:
https://github.com/gnosis/safe-config-service/pull/363/commits/3be622b83cd22b5f90557229b34f888774426e11